command/hcl2_upgrade.go

```diff
@@ -771,6 +778,9 @@ func (p *LocalsParser) Write(out *bytes.Buffer) {
 		}
 		fmt.Fprintln(out, `locals { timestamp = regex_replace(timestamp(), "[- TZ:]", "") }`)
 	}
+	if isotime {
+		fmt.Fprintln(out, `# The "legacy_isotime" function has been provided for backwards compatability, but we recommend switching to the timestamp and formatdate functions.`)
+	}
```

Comment:
I think it would be nice to add an example test/case in the hcl2_formatted tests: 
there: https://github.com/hashicorp/packer/blob/610dde7f02327b6c37a480b7074e07cc09e64814/command/test-fixtures/hcl2_upgrade/complete/expected.pkr.hcl

Reply:
+1 on @azr suggestion

Reply:
👍

Reply:
Turns out the tests that were failing were in this area -- isotime was already included in them so I had to change the expected output 🎉